plugins/okta/services/okta.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+/**
+ * okta.js service
+ *
+ * @description: A set of functions similar to controller's actions to avoid code duplication.
+ */
+
+module.exports = {
```

Comment:
we're not exporting anything. is this file needed?

Reply:
Not needed currently but comes as a part of boilerplate.

Reply:
let's remove everything that's not needed to reduce clutter